packages/ml/matlab/CMakeLists.txt

```diff
@@ -53,18 +53,22 @@ if(TPL_ENABLE_MATLAB)
 
   # Manually drop in options from the mex script (R2009b) on a linux platform.
   # g++ -O -pthread -shared -Wl,--version-script,/usr/local/matlab/7.9/extern/lib/glnxa64/mexFunction.map -Wl,--no-undefined -o  "mlmex.mexa64"   "mlmex-mlmex.o"  -lm -Wl,-rpath-link,/usr/local/matlab/7.9/bin/glnxa64 -L/usr/local/matlab/7.9/bin/glnxa64 -lmx -lmex -lmat -lm
-  SET(MLMEX_OPTS1 "-pthread;-shared;-Wl,--version-script,${MATLAB_ROOT}/extern/lib/${MATLAB_ARCH}/mexFunction.map;-Wl,--no-undefined")
-  
-  SET(MLMEX_OPTS2 "-Wl,-rpath-link,${MATLAB_ROOT}/bin/${MATLAB_ARCH} ${${PROJECT_NAME}_EXTRA_LINK_FLAGS}")
-    
+  if (NOT APPLE)
+    SET(MLMEX_OPTS1 "-pthread;-shared;-Wl,--version-script,${MATLAB_ROOT}/extern/lib/${MATLAB_ARCH}/mexFunction.map;-Wl,--no-undefined")
+    SET(MLMEX_OPTS2 "-Wl,-rpath-link,${MATLAB_ROOT}/bin/${MATLAB_ARCH}")
```

Comment:
@jgedicke Why have the options for !Apple (i.e., Linux) changed?  `${${PROJECT_NAME}_EXTRA_LINK_FLAGS}` is missing.

Reply:
The inclusion of `${${PROJECT_NAME}_EXTRA_LINK_FLAGS}` is platform independent and not specific to mex either, hence they should not be part of `MLMEX_OPTS` , therefore they are now included directly in line 71 below.